api/views.py

```diff
@@ -21,6 +22,10 @@ class ItemViewSet(ModelViewSet):
     http_method_names = ['get']
 
     def get_queryset(self):
-
         user = self.request.user
         return Item.objects.filter(assigned_to=user)
+
+    def get_object(self):
+        queryset = self.get_queryset()
```

Comment:
Consider refactoring this to `queryset = Item.objects.filter(assigned_to=self.request.user)`. Otherwise looks good overall.